docs/application/dotnet/guides/opentk/window-of-opentk.md

```diff
@@ -1,26 +1,24 @@
 # Window of OpenTK
 
-In the OpenTK app, `OpenTKGameApplication` creates a `Window` after `OnCreate` method is called. `Window` is a
-property of `TizenGameApplication`. This document will describe how to use this `Window` in your OpenTK app.
+This guide explains how to use `Window` property of `OpenTKGameApplication` class created by `OnCreate()` method in your OpenTK app.
```

Comment:
1.  please change OpenTKGameApplicaiton to TizenGameApplication, the name is wrong, my mistake.
2. Window is not created by OnCreated() method, it is created by internal method, this internal method is triggered by 'OnCreated' method, therefore, it would be avaiable util 'OnCreate' method is called.

    May be this sentence could be change to :
    This guide explains how to use `Window` property of `TizenGameApplication` class, `Window` would be available util OnCreate() method is called in your OpenTK app.

Reply:
I think the following sentence is better.

I think "`Window` would be available util OnCreate() method is called in your OpenTK app" is not different meaning that you want.

This guide explains how to use `Window` property of `TizenGameApplication` class. `Window` is 
 internally created in `OnCreate()` method of the base class.